### PR TITLE
Update genesis blocks - Closes #674

### LINKED ISSUE #674: Update genesis blocks
### Description

Update genesis blocks for devnet and alphanet.

### Acceptance Criteria

- The devnet and alphanet genesis blocks are recreated with the following ensured:
  - Legacy accounts are re-generated with known passphrases
  - All the validators are re-generated with known passphrases
  - The genesis blocks are recreated with updated assets

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #674 

### How was it solved?

- [x] The devnet and alphanet  genesis blocks are recreated with the following ensured:
  - [x] Legacy accounts are re-generated with known passphrases
  - [x] All the validators are re-generated with known passphrases
  - [x] The genesis blocks are recreated with updated assets
- [x] Fix issue with legacy endpoint
- [x] Fix issue with reclaim command
- [x] Update reclaim command name to reclaimLSK
- [x] Fix unit tests 

### How was it tested?
Local

### CODE REVIEW

File: config/alphanet/config.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"system": {
-		"dataPath": "~/.lisk"
+		"dataPath": "~/.lisk",
+		"keepEventsForHeights": 300
 	},
 	"rpc": {
 		"modes": ["ipc"]
```
Comment (sameersubudhi) [inline config/alphanet/config.json:7]:
```suggestion
		"modes": ["ipc"],
		"port": 7887,
		"host": "127.0.0.1"
```

File: config/devnet/config.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
 	"system": {
-		"dataPath": "~/.lisk"
+		"dataPath": "~/.lisk",
+		"keepEventsForHeights": 300
 	},
 	"rpc": {
 		"modes": ["ipc"]
```
Comment (sameersubudhi) [inline config/devnet/config.json:7]:
```suggestion
		"modes": ["ipc"],
		"port": 7887,
		"host": "127.0.0.1"
```

File: src/application/modules/legacy/endpoint.ts [hunk 3]
```diff
@@ -41,8 +38,8 @@ export class LegacyEndpoint extends BaseEndpoint {
 		const legacyStore = this.stores.get(LegacyAccountStore);
 
 		try {
-			const legacyAddress = getLegacyAddressFromPublicKey(publicKey);
-			const hasLegacyAddress = await legacyStore.has(ctx, publicKey);
+			const legacyAddress = getLegacyAddress(publicKey).toString('hex');
+			const hasLegacyAddress = await legacyStore.has(ctx, Buffer.from(legacyAddress, 'hex'));
```
Comment (sameersubudhi) [inline src/application/modules/legacy/endpoint.ts:42]:
```suggestion
			const legacyAddressBuffer = getLegacyAddress(publicKey);
			const legacyAddress = legacyAddressBuffer.toString('hex');
			const hasLegacyAddress = await legacyStore.has(ctx, legacyAddressBuffer);
```

File: test/application/modules/legacy/commands/reclaim.spec.ts [hunk 4]
```diff
@@ -25,11 +25,12 @@ import {
 } from '../../../../../src/application/modules/legacy/schemas';
 
 const {
-	legacyAddress: { getLegacyAddressFromPublicKey },
+	legacyAddress: { getFirstEightBytesReversed },
+	utils: { hash },
 } = cryptography;
 
 const getLegacyAddress = (publicKey): any => {
```
Comment (sameersubudhi) [inline test/application/modules/legacy/commands/reclaim.spec.ts:32]: Please import the method from `src/application/modules/legacy/utils.ts` and use it. Let us avoid duplication of logic.

File: test/application/modules/legacy/endpoint.spec.ts [hunk 5]
```diff
@@ -93,11 +93,13 @@ describe('LegacyEndpoint', () => {
 			});
 
 			const expectedLegacyAccount = {
-				legacyAddress: getLegacyAddressFromPassphrase(accounts.existingAccount.passphrase),
+				legacyAddress: getFirstEightBytesReversed(hash(existingPublicKey)).toString('hex'),
```
Comment (sameersubudhi) [inline test/application/modules/legacy/endpoint.spec.ts:96]: Same as above.

File: test/application/modules/legacy/endpoint.spec.ts [hunk 6]
```diff
@@ -93,11 +93,13 @@ describe('LegacyEndpoint', () => {
...
 				balance: '1000',
 			};
 
-			when(mockStoreHas).calledWith(existingPublicKey).mockReturnValue(true);
+			when(mockStoreHas)
+				.calledWith(Buffer.from(expectedLegacyAccount.legacyAddress, 'hex'))
```
Comment (sameersubudhi) [inline test/application/modules/legacy/endpoint.spec.ts:101]: Let us store `Buffer.from(expectedLegacyAccount.legacyAddress, 'hex')` in a constant and reuse it.

File: test/application/modules/legacy/endpoint.spec.ts [hunk 7]
```diff
@@ -119,17 +121,22 @@ describe('LegacyEndpoint', () => {
 				},
 			});
 
-			const legacyAddress = getLegacyAddressFromPublicKey(nonExistingPublicKey);
+			const expectedLegacyAccount = {
+				legacyAddress: getFirstEightBytesReversed(hash(nonExistingPublicKey)).toString('hex'),
```
Comment (sameersubudhi) [inline test/application/modules/legacy/endpoint.spec.ts:125]: Same as above.

File: test/scripts/legacyAccounts.ts [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+	const legacyAddress = cryptography.legacyAddress.getLegacyAddressFromPassphrase(passphrase);
+	const firstEightBytesReversed = cryptography.legacyAddress.getFirstEightBytesReversed(
+		cryptography.utils.hash(publicKey),
+	);
+	return {
+		passphrase,
+		privateKey: privateKey.toString('hex'),
+		publicKey: publicKey.toString('hex'),
+		address: address.toString('hex'),
+		legacyAddress: legacyAddress.toString(),
```
Comment (sameersubudhi) [inline test/scripts/legacyAccounts.ts:33]: `toString()` is redundant here.
```suggestion
		legacyAddress: legacyAddress,
```

File: test/scripts/validators.ts [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
```
Comment (sameersubudhi) [inline test/scripts/validators.ts:15]:
```suggestion
```

File: test/scripts/validators.ts [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+	const passphrases: any = [];
+	const encryptedMessageObject = {};
+
+	for (let i = 0; i < count; i += 1) {
+		const passphrase = Mnemonic.generateMnemonic(256);
+		passphrases.push(passphrase);
+
+		const accountKeyPath = `m/44'/134'/${offset}'`;
+		const generatorKeyPath = `m/25519'/134'/${offset}'/${offset}'`;
+		const blsKeyPath = `m/12381/134/${offset}/${offset}`;
```
Comment (sameersubudhi) [inline test/scripts/validators.ts:33]:
```suggestion
	const keys: any = [];
	const passphrases: any = [];
	const encryptedMessageObject = {};

	for (let i = 0; i < count; i += 1) {
		const passphrase = Mnemonic.generateMnemonic(256);
		passphrases.push(passphrase);

		const accountKeyPath = `m/44'/134'/0'`;
		const generatorKeyPath = `m/25519'/134'/0'/0'`;
		const blsKeyPath = `m/12381/134/0/0`;
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: src/application/modules/legacy/commands/reclaim.ts [hunk 11]
```diff
@@ -27,20 +27,17 @@ import { ADDRESS_LEGACY_RESERVE } from '../constants';
 
 import { reclaimParamsSchema } from '../schemas';
 import { ReclaimParamsData, TokenIDReclaim } from '../types';
```
Comment (sameersubudhi) [inline src/application/modules/legacy/commands/reclaim.ts:29]: Please rename the schema and params to be consistent with the `reclaimLSK` command too.

**Review by sameersubudhi — APPROVED** [review]
Works well! Nice job @nagdahimanshu! 🚀

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**